src/server/game/DungeonFinding/LFGMgr.h

```diff
@@ -128,6 +128,11 @@ enum LfgRoleCheckState
     LFG_ROLECHECK_NO_ROLE                        = 6       // Someone selected no role
 };
 
+enum LfgRoleClasses {
+    TANK   = 0b10000100011, // warrior, paladin, dk, druid
```

Comment:
is there a way for a normal person to know what does this number mean ? some other enum defined somewhere with different classes ?

Reply:
Good point, i'll do `(1<<(CLASS_WARRIOR-1)) | 1<<(CLASS_PALADIN-1)) ...`. Don't think there's any mask-ready enum, and that's the style used by other `SharedDefines.h` stuff.